### DISCUSSION TITLE
File upload for github education

### DISCUSSION THREAD

**Discussion by Do-Not-Dusturb** [community/community#141903] [GitHub Education]
### Hi everyone,

The first time I applied for github education as a student, that was at home, now I am trying again at school and before I was able to upload proof of  my school, but now its asking for a picture, I don't have my proof in my hands physically, I have it as a computer file. What do I do please help.

**Comment by sensie01** [comment]
Hello, to address your issue with the GitHub Education application process, you have a few potential options:

**1-Use the computer file**: If you have the proof as a digital file on your computer, you might be able to upload that directly. Look for an option to upload a file rather than take a picture.
**2-Take a picture of your screen**:If the system only allows photos, you could try taking a picture of your computer screen displaying the digital proof using your phone or another device.
**3- or Contact GitHub Support**:If none of the above options work, you should reach out to GitHub Education support. They may be able to provide an alternative method for submitting your proof or assist you with your specific situation.
**4- Alternatively, try a different device or browser**:Sometimes, the upload options can vary depending on the device or browser you're using. Try accessing the application from a different device or browser to see if you get the file upload option.
**Check for alternative verification methods**:GitHub might offer other ways to verify your student status, such as using your school email address or connecting to your school's network.

**Reply by Do-Not-Dusturb** [reply]
No, I just said I cant upload there is no option

**Comment by MG72021** [comment]
[google.com](url)

**Reply by Do-Not-Dusturb** [reply]
?
